.travis.yml

```diff
@@ -1,6 +1,8 @@
+dist: trusty
+sudo: false
```

Comment:
Is it caused by the changes described in [this blog post|https://blog.travis-ci.com/2017-05-04-precise-image-updates]?